tasks-by-users/Tauqeer-Ahmed-99/03. JS Monster Game/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<head>
```

Comment:
This project folder has 03. Dot .......Others ones have 02- dash. Consistency is missing. 
No space in folders. Lowercase folder name please